public_html/wp-content/plugins/wordcamp-remote-css/libraries/adhocore-php-jwt/JWT.php

```diff
@@ -0,0 +1,273 @@
+<?php
```

Comment:
Can this be added to our `composer.json`, instead of bundling it manually?

That'd help keep it updated, get informed about security releases, etc.

Reply:
Yep! I don't actually know how composer is being used in the WordCamp code (or realised that it was there at all) so I'll take a look at that.

Reply:
Done in c71bd01

Reply:
Looks great 👍